darwin/client.py

```diff
@@ -999,7 +1001,8 @@ def default_base_url() -> str:
         return os.getenv("DARWIN_BASE_URL", "https://darwin.v7labs.com")
 
     def _get_headers(self, team_slug: Optional[str] = None) -> Dict[str, str]:
-        headers: Dict[str, str] = {"Content-Type": "application/json"}
+        headers: Dict[str, str] = {"Content-Type": "application/json", "X-Darwin-Payload-Compression-Version": "1"}
```

Comment:
Should we set this even if later we don't actually compress? It feels this should be set based on the same condition.